controller/app.controller.js

```diff
@@ -44,3 +44,18 @@ exports.getCommentsByArticleId = (req,res,next) => {
    .catch (next);
  }
 
+//Task 7 POST /api/articles/:article_id/comments
+exports.addComment = (req,res,next) => {
+   const article_id = req.params.article_id
+   const newComment = req.body;
+   const promises = [AddCommentByArticleId(article_id,newComment)];
+   if (article_id ) {
+       promises.push(checkExists('articles','article_id',article_id));
+   }
+   Promise.all(promises)
+   .then ((resolvedPromises) => {
+      const comments = resolvedPromises[0]
+      res.status(201).send({comments:comments[0]})
```

Comment:
As we are only posting a new comment and not comments, could we change this key to 'comment' please?

Reply:
done